pkg/common/cns-lib/volume/manager.go

```diff
@@ -690,3 +692,26 @@ func (m *defaultManager) QueryVolumeInfo(ctx context.Context, volumeIDList []cns
 	log.Infof("QueryVolumeInfo successfully returned volumeInfo volumeIDList %v:, opId: %q", volumeIDList, taskInfo.ActivationId)
 	return volumeInfoResult, nil
 }
+
+func (m *defaultManager) RelocateVolume(ctx context.Context, relocateSpecList ...cnstypes.BaseCnsVolumeRelocateSpec) (*object.Task, error) {
+	log := logger.GetLogger(ctx)
+	err := validateManager(ctx, m)
+	if err != nil {
+		log.Errorf("validateManager failed with err: %+v", err)
+		return nil, err
+	}
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Can you use the original context supplied in func argument `RelocateVolume(ctx context.Context,`?